@kiva/kv-components/vue/KvCommentsListItem.vue

```diff
@@ -149,10 +151,14 @@ export default {
 
 		const showInput = ref(false);
 		const commentsAddRef = ref(null);
+		const authorInfo = ref();
 
 		const commentText = computed(() => comment?.value?.data?.text ?? '');
-		const authorImage = computed(() => comment?.value?.user?.data?.image ?? '');
-		const authorName = computed(() => comment?.value?.user?.data?.name ?? '');
+
+		const authorId = computed(() => comment?.value?.user?.data?.publicLenderId ?? '');
+		const authorImage = computed(() => authorInfo?.value?.image?.url ?? '');
```

Comment:
This could be a new ticket, but I think we should add a loader while this new data is loading, it could be one loader or a little circle and block for the name text.

Reply:
Totally agreed! Looks like don't needed locally, but let's see how works on dev. At least having this as a package update...